binding/web/src/rhino.ts

```diff
@@ -142,52 +176,247 @@ export class Rhino implements RhinoEngine {
 
     this._wasmMemory = handleWasm.memory;
     this._pvFree = handleWasm.pvFree;
-    this._memoryBuffer = new Int16Array(handleWasm.memory.buffer);
-    this._memoryBufferUint8 = new Uint8Array(handleWasm.memory.buffer);
-    this._memoryBufferView = new DataView(handleWasm.memory.buffer);
-
-    this._objectAddress = handleWasm.objectAddress;
     this._contextAddress = handleWasm.contextAddress;
     this._inputBufferAddress = handleWasm.inputBufferAddress;
+    this._intentAddressAddress = handleWasm.intentAddressAddress;
     this._isFinalizedAddress = handleWasm.isFinalizedAddress;
     this._isUnderstoodAddress = handleWasm.isUnderstoodAddress;
-    this._intentAddressAddress = handleWasm.intentAddressAddress;
     this._numSlotsAddress = handleWasm.numSlotsAddress;
+    this._objectAddress = handleWasm.objectAddress;
     this._slotsAddressAddressAddress = handleWasm.slotsAddressAddressAddress;
     this._valuesAddressAddressAddress = handleWasm.valuesAddressAddressAddress;
 
+    this._memoryBuffer = new Int16Array(handleWasm.memory.buffer);
+    this._memoryBufferUint8 = new Uint8Array(handleWasm.memory.buffer);
+    this._memoryBufferView = new DataView(handleWasm.memory.buffer);
     this._processMutex = new Mutex();
+
+    this._inferenceCallback = inferenceCallback;
+    this._processErrorCallback = processErrorCallback;
   }
 
   /**
-   * Releases resources acquired by WebAssembly module.
+   * Get Rhino engine version.
    */
-  public async release(): Promise<void> {
-    await this._pvRhinoDelete(this._objectAddress);
-    await this._pvFree(this._contextAddress);
-    await this._pvFree(this._inputBufferAddress)
-    await this._pvFree(this._isFinalizedAddress)
-    await this._pvFree(this._isUnderstoodAddress)
-    await this._pvFree(this._intentAddressAddress)
-    await this._pvFree(this._numSlotsAddress)
-    await this._pvFree(this._slotsAddressAddressAddress)
-    await this._pvFree(this._valuesAddressAddressAddress)
+  get version(): string {
+    return Rhino._version;
+  }
+
+  /**
+   * Get frame length.
+   */
+  get frameLength(): number {
+    return Rhino._frameLength;
+  }
+
+  /**
+   * Get sample rate.
+   */
+  get sampleRate(): number {
+    return Rhino._sampleRate;
+  }
+
+  /**
+   * Get context info.
+   */
+  get contextInfo(): string {
+    return Rhino._contextInfo;
+  }
+
+  /**
+   * Creates an instance of the Rhino speech-to-intent engine using a base64'd string
```

Comment:
I know it's pedantic, but Dilek is very specific about the branding
```suggestion
   * Creates an instance of the Rhino Speech-to-Intent engine using a base64'd string
```